script/Arcueid/Day 4/04_00_ARC04_6_2B.txt

```diff
@@ -282,47 +291,51 @@ Although their body has surpassed that of a human, think of them as commanders o
 [sha:1c16a41e104a78c051b19f75ab908f4b28a7a606]{
 -- Page 492, Offset 6915.
 -- 「あれはもう一っこ下、Ⅱ階梯目の『屍鬼』。
-"Those things are one rank below. A Rank II %{i}Ghoul%{/i}.
+"Those things are one tier below. Rank II %{i}Ghouls%{/i}.
+//generally Rank should be captalized to emphasise its not just any ol' rank but the vampire ranks
 }
 [sha:053c212fcbc48a7d62a4832f66f1a6aa8099af43]{
 -- Page 492, Offset 6916.
 -- 人間への擬態もできない、命じられた事をこなすだけの労働力よ」
-Vampires that are unable to mimic humans, and are workers that simply do as they're told."
+They're basically just workers, and can't do much more than follow orders, let alone pass off as human."
+//i like this sentence structure better
 }
 [sha:92d34aaf4a94847b13b90a47eb97e817f3569a64]{
 -- Page 493, Offset 6917.
 -- 興が乗ってきたのか、アルクェイドは吸血鬼たちの段階を図で説明しだした。
-Arcueid seems to have warmed up to the topic, and is illustrating the various ranks of vampire.
+Arcueid seems to have gotten really into it, and starts explaining the various vampire stages using a diagram.
```

Comment:
'stages' sounds like they progress, which iirc they don't (at least for the low ranks?) - maybe just 'various _kinds_ of vampires' or whatever